core/src/edu/asu/c3simulator/screens/FundingScreen.java

```diff
@@ -49,29 +50,27 @@ public void clicked(InputEvent event, float x, float y)
 				- (0.01f * stage.getWidth()), (0.01f * stage.getWidth()));
 		
 		// TODO add Corner Advisor
-		// TODO add Home Button
+		HomeButton home = new HomeButton(game);
 		
-		NavigationPanel navigationPanel = createNavigationPanel();
-		
-		stage.addActor(navigationPanel);
+		stage.addActor(home);
 		stage.addActor(continueButton);
 		stage.addActor(ScreenDiscription);
 	}
 	
 	@Override
-	public NavigationPanel createNavigationPanel()
+	public void createNavigationPanel()
```

Comment:
Actually, the solutions you proposed are all used in the code. The "createNavigationPanel" has exactly the same functionality as the "initialize" method you and justin created. it is just a different name. Also, all navigation panels take some AllManagementScreens instance. The problem that we encountered in our previous demo was that we tried to reference an AllManagementScreens instance that wasn't initialized yet which for some reason resulted in it being considered null even after initialization.